flyway/src/main/scala/com/avast/sst/flyway/FlywayModule.scala

```diff
@@ -0,0 +1,40 @@
+package com.avast.sst.flyway
+
+import cats.effect.Sync
+import org.flywaydb.core.Flyway
+
+object FlywayModule {
+
+  @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+  def make[F[_]: Sync](config: FlywayConfig): F[Flyway] = {
+    Sync[F].delay {
+      val builder = Flyway
+        .configure
```

Comment:
just for info: in case the fluent config turns out cumbersome, there's still a [classical one](https://flywaydb.org/documentation/api/javadoc/org/flywaydb/core/api/configuration/ClassicConfiguration.html)